src/selectize.js

```diff
@@ -1466,6 +1516,18 @@ $.extend(Selectize.prototype, {
 		return this.getElementWithValue(value, this.$control.children());
 	},
 
+	/**
+	 * Returns the jQuery element of the item
+	 * matching the given textContent.
+	 *
+	 * @param {string} value
+	 * @param {boolean} ignoreCase
+	 * @returns {object}
+	 */
+	getFirstItemMatchedByTextContent: function(textContent, ignoreCase = false) {
```

Comment:
remove  '= false'  from the ignoreCase argument, and instead check and initialize the default inside the method body